cscs-checks/tools/profiling_and_debugging/notool.py

```diff
@@ -46,43 +47,44 @@ def __init__(self, lang):
             'PrgEnv-pgi': {'version': 201307},
         })
         # a scopedict is better than this:
+        self.language = lang
         # if (self.language == 'C++' and
         #    self.current_environ.name == 'PrgEnv-pgi'):
         #    self.omp_versions['PrgEnv-pgi'] = '200805'
-
+        self.maintainers = ['JG', 'MK']
+        self.tags = {'production'}
+        if self.current_system.name in {'dom', 'daint'}:
+            self.post_run = ['module list -t']
```

Comment:
Why are you listing the modules as a post run command and not having any sanity check associated to it? Is there anything we should be aware of? Or it is just to "save" the modules for later check?

Reply:
https://github.com/eth-cscs/reframe/pull/804/commits/4942a8c52b8428aed33e5f98392ef7c51aabcea0

Reply:
Based on our offline discussion, then you just want to save the modules for later check and historical check. In this case, there is no need for additional sanity checks.